Signal/src/call/CallAudioService.swift

```diff
@@ -102,42 +102,6 @@ protocol CallAudioServiceDelegate: class {
         }
     }
 
-    class Sound: NSObject {
-
-        static let incomingRing = Sound(filePath: "r", fileExtension: "caf", loop: true)
-        static let outgoingRing = Sound(filePath: "outring", fileExtension: "mp3", loop: true)
-        static let dialing = Sound(filePath: "sonarping", fileExtension: "mp3", loop: true)
-        static let busy = Sound(filePath: "busy", fileExtension: "mp3", loop: false)
-        static let failure = Sound(filePath: "failure", fileExtension: "mp3", loop: false)
```

Comment:
Superceded by the more general `OWSSounds`/`OWSSound`.  Maybe I should rename `OWSSounds` to `OWSSoundManager`.